Source/NSURL+Kakapo.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  NSURL+Kakapo.swift
+//  KakapoExample
+//
+//  Created by Joan Romano on 31/03/16.
+//  Copyright © 2016 devlucky. All rights reserved.
+//
+
+import Foundation
+
+typealias URLInfo = (params: [String : String], queryParams: [String : String])
+
+/// parseUrl: Checks and parses if a given `handleURL` representation matches a `requestURL`. Examples:
```

Comment:
Using this http://ericasadun.com/2015/06/14/swift-header-documentation-in-xcode-7/